docs/user_guide/analytics.rst

```diff
@@ -1,6 +1,35 @@
 Analytics and usage services
 ============================
 
+The theme supports analytics via the ``analytics`` option. It is configured
+by passing a dictionary with options. See the sections bellow for relevant
+options depending on the analytic provider that you want to use.
+
+.. code:: python
+
+   html_theme_options = {
+       "analytics": analytics_options,
+   }
+
+Plausible Analytics (recommended)
+=================================
+
+https://plausible.io can be used to gather simple
+and privacy-friendly analytics for the site. The configuration consists in
+a server URL and a specific domain. Plausible' javascript will be included in
+all html pages to gather metrics. And the dashboard can be accessed at
+``https://url/my_domain``.
+
+The Scientific-Python community can offer a self-hosted server. Contact the
```

Comment:
Is this documented somewhere? We were looking into Plausible for Jupyter a while back, but didn't do it in-part because it was pretty expensive to run at the scale of Jupyter's website. I wonder if we could piggy-back on scientific-python with this?

Reply:
Documented as what is the process to get access? Nothing yet, this would be the first document talking about that. At some point we might have some mention on our website when we list all the initiatives we have.

@stefanv maybe you can answer this one? I don't know how hard this is for the server.
But I would guess it's ok. SciPy is getting 100k per month which is nothing. I don't know how many RPS the server can handle but I would assume >100 RPS (based on my personal experience, we should see this with e.g. locust to be sure)